features/step_definitions/email_custom_steps.rb

```diff
@@ -40,8 +40,9 @@
 end
 
 Then /^the email should say what time it was sent$/ do
-  # mailer time format: 11:38PM EDT Thu 29 September 2016
-  nowish = Time.now.strftime('%I:%M%p %Z %a %d %B %Y')
+  # Rails default: https://github.com/rails/rails/blob/v7.0.2.3/activesupport/lib/active_support/locale/en.yml#L25
+  # But truncated to the day so the test don't need to run under a second to pass
+  nowish = Time.now.utc.strftime('%a, %d %b %Y')
   step %{the email should contain "Sent at #{nowish}"}
   step %{the email should not contain "sent_at"}
```

Comment:
I'm wondering if that step shouldn't simply be removed instead of maintained. It's only used in a single test, and that test might not be relevant anymore, as the original [commit](https://github.com/otwcode/otwarchive/commit/40bce3aa59b799d5f6724d80f88197e08dc8bb85) references timezones, and whether timezones appear or not in the text is now dependent on Rails' default localization (cf the example, where they appear in the English version but not the French one).

Reply:
Yeah that makes sense. Let's remove the step. For the test, I think you can freeze time at the start and do a simple "should see" check for the time string, however it's formatted right now. As an example:

https://github.com/otwcode/otwarchive/blob/1f8558452e6984a41ace47074324013d2a3454f8/features/other_a/reading.feature#L29-L34

Reply:
https://github.com/otwcode/otwarchive/pull/4168/commits/23ed696266a8f846f21f441eb98a94d0af942aac